OpenTyping/InternetServices.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.NetworkInformation;
+
+namespace OpenTyping
+{
+    class InternetServices
+    {
+        public static bool IsConnected()
+        {
+            try
+            {
+                Ping myPing = new Ping();
+                String host = "google.com";
+                byte[] buffer = new byte[32];
+                int timeout = 1000;
+                PingOptions pingOptions = new PingOptions();
+                PingReply reply = myPing.Send(host, timeout, buffer, pingOptions);
+                return (reply.Status == IPStatus.Success);
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+        }
+        public static void SendLogsToServer()
+        {
+            using (WebClient client = new WebClient())
```

Comment:
You'd better using the using syntax in the head of code file like importing library.

Reply:
This 'using' statement is different than the one we use for namespaces. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement

Reply:
Thanks. I understood the meaning of the usage of using syntax. But when I read the article below, this using syntax doesn't look like working as we expected. So how about using the HttpClient without using syntax instead of WebClient? https://www.youtube.com/watch?v=MdrSEcQtJDY&list=RDCMUCqCqf21juyyL_8peGs2CWGg&index=42, https://www.thecodebuzz.com/using-httpclient-best-practices-and-anti-patterns/